FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -618,6 +622,51 @@ - (BOOL)insertOrUpdatePersonalizationConfig:(NSDictionary *)dataValue
   return YES;
 }
 
+- (void)insertOrUpdateRolloutTableWithKey:(NSString *)key
+                                    value:(NSArray *)value
+                        completionHandler:(RCNDBCompletion)handler {
+  dispatch_async(_databaseOperationQueue, ^{
+    BOOL success = [self insertOrUpdateRolloutTableWithKey:key value:value];
+    if (handler) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        handler(success, nil);
+      });
+    }
+  });
+}
+
+- (BOOL)insertOrUpdateRolloutTableWithKey:(NSString *)key value:(NSArray *)arrayValue {
+  RCN_MUST_NOT_BE_MAIN_THREAD();
+  NSError *error;
+  NSData *dataValue = [NSJSONSerialization dataWithJSONObject:arrayValue
```

Comment:
Is `arrayValue` here referring to each rollout entry?

Reply:
A list of rollout metadata (repeated proto field in the fetch response).